utilities/oadp.py

```
@@ -28,19 +36,55 @@
+            "Deleting Velero resource",
+            extra={
+                "resource_kind": resource.kind,
+                "resource_name": resource.name,
+                "command": " ".join(command),
+            },
+        )
+
+        velero_pod.execute(command=command)
+
+    except ResourceNotFoundError:
+        LOGGER.error(
```

Ruff (0.14.11)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)